docs/production/export-and-import.md

```diff
@@ -7,42 +7,54 @@ service (or back):
     version (most precisely, one where `manage.py showmigrations` has
-    the same output).
+    identical output).
   - Backups must be restored on a server running the same PostgreSQL
     version. To install Zulip with the same version of PostgreSQL that
     the backup was taken on, pass the desired version with [the
     `--postgresql-version` argument][installer-options] when
-    installing.
+    installing. Note that PostgreSQL is easy to [upgrade
+    independently][postgres-upgrade] from the rest of your Zulip
+    installation.
   - Backups aren't useful for migrating organizations between
     self-hosting and Zulip Cloud (which may require renumbering all
     the users/messages/etc.).
 
-  We highly recommend this tool in situations where it is applicable,
-  because it is highly optimized and highly stable, since the hard
-  work is done by the built-in backup feature of PostgreSQL. We also
-  document [backup details](#backup-details) for users managing
-  backups manually.
+  We also document [backup details](#backup-details) for users
+  managing backups manually.
 
-- The logical [Data export](#data-export) tool is designed for
+- The [logical data export](#data-export) tool is designed for
   migrating data between Zulip Cloud and other Zulip servers, as well
-  as various auditing purposes. The logical export tool produces a
-  `.tar.gz` archive with most of the Zulip database data encoded in
-  JSON files–a format shared by our [data
-  import](#import-into-a-new-zulip-server) tools for third-party
-  services like
-  [Slack](https://zulip.com/help/import-from-slack).
-
-  Like the backup tool, logical data exports must be imported on a
-  Zulip server running the same version. However, logical data
-  exports can be imported on Zulip servers running a different
-  PostgreSQL version or hosting a different set of Zulip
-  organizations. We recommend this tool in cases where the backup
-  tool isn't applicable, including situations where an easily
-  machine-parsable export format is desired.
+  as various auditing purposes.
+
+  We recommend this tool in cases where the backup tool isn't
+  applicable, including situations where an easily machine-parsable
+  export format is desired. This tool has a few limitations and
+  caveats:
+
+  - Like the backup tool, logical data exports must be imported on a
+    Zulip server running the same Zulip version. However, logical data
+    exports can be imported on Zulip servers running a different
+    PostgreSQL version or hosting a different set of Zulip
+    organizations.
+  - The data export tool requires extra care to avoid disconnecting
+    your Zulip server from its [Mobile Push Notifications Service][mobile-push]
+    registration, as detailed in the data import documentation.
+  - Transferring an organization via the data export tool results in
+    significant user-facing disruption, such as logging all users out of
+    their accounts.
```

Comment:
And requiring password resets, it's probably worth noting as well.

Reply:
Added.